extensions/scarb-snforge-test-collector/src/main.rs

```diff
@@ -52,7 +52,7 @@ fn main() -> Result<()> {
 
         // artifact saved to `{target_dir}/{profile_name}/{package_name}.sierra.json`
         let output_path =
-            snforge_target_dir.join(&format!("{}.snforge_sierra.json", package_metadata.name));
+            snforge_target_dir.join(format!("{}.snforge_sierra.json", package_metadata.name));
```

Comment:
unrelated change?

Reply:
CI fails because of this, not sure why. I did a rebase to main, and it still failed: https://github.com/software-mansion/scarb/actions/runs/10735766063/job/29773752730